File: P1A5 Creacion de Clases/Fecha - main.py
class Fecha:
    def __init__(self):
        self.unMes = 0
        self.unDia = 0
        self.unAnio = 0
        
    def mostrarFecha(self):
        return f"{self.unMes}/{self.unDia}/{self.unAnio}"

  
fechaActual = Fecha()

print("Ingrese los siguientes valores en numeros")
fechaActual.unDia = int(input("Que dia es hoy: "))
fechaActual.unMes = int(input("En que mes estamos: "))
fechaActual.unAnio = int(input("En que anio estamos: "))

print("\n--- FECHA ACTUAL ---")
fechaActual.mostrarFecha()

File: P1A5 Creacion de Clases/Fecha - main.cs
using System;
class Fecha{
    public int unMes;
    public int unDia;
    public int unAnio;
    
    public Fecha(){
        unMes = 0;
        unDia = 0;
        unAnio = 0;
    }
    
    public string mostrarFecha(){
        return $"{unMes}/{unDia}/{unAnio}";
    }
}

class FechaTest{
    static void Main (String[] args){
        Fecha fechaActual = new Fecha();
        
        Console.WriteLine("Ingrese los siguientes valores en numeros");
        
        Console.WriteLine("Que dia es hoy: ");
        fechaActual.unDia = Convert.ToInt32(Console.ReadLine());
        
        Console.WriteLine("En que mes estamos: ");
        fechaActual.unMes = Convert.ToInt32(Console.ReadLine());
        
        Console.WriteLine("En que anio estamos: ");
        fechaActual.unAnio = Convert.ToInt32(Console.ReadLine());
        
        Console.WriteLine("\n--- FECHA ACTUAL ---");
        Console.WriteLine(fechaActual.mostrarFecha());
    }
}